tests/components/homekit/test_type_sensors.py

```diff
@@ -1,7 +1,7 @@
 """Test different accessory types: Sensors."""
+from homeassistant.components.binary_sensor import DEVICE_CLASS_MOTION
```

Comment:
If you are going to do the tests at the same time (I thought you were doing them separately?)....

...then this line can be removed and the references to it below can be be `BinarySensorDeviceClass.MOTION`.

(I realise that the the state variable wants a string, but I am working on the assumption that `DEVICE_CLASS_MOTION` will disappear at the end of the deprecation period.)

Reply:
The tests should be updated as well, but that can be another PR